examples/multipole/multipole.yml

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env yason.py
+temperature: 300
+random: {seed: hardware}
+geometry: {type: sphere, radius: 10}
+mcloop: {macro: 10, micro: 200000}
+atomlist:
+    - pos1: {q: 1.0, sigma: 4}
+    - pos2: {q: 2.0, sigma: 4}
+    - pos3: {q: 3.0, sigma: 4}
+    - neg1: {q: -1.0, sigma: 4}
+    - neg2: {q: -2.0, sigma: 4}
+    - neg3: {q: -3.0, sigma: 4}
+moleculelist:
+    - monopole:
+        rigid: true
+        insdir: [0,0,0]
+        structure:
+            - neg2: [0,0,0]
+    - dipole:
+        rigid: true
+        insdir: [0,0,1]
+        structure:
+            - neg2: [-0.5, 0, 0]
+            - pos3: [0.5, 0, 0]
+    - quadrupole:
+        rigid: true
+        insdir: [0,0,1]
+        structure:
+            - pos1: [1.0, 1.0, 0.0]
+            - pos1: [-1.0, -1.0, 0.0]
+            - neg1: [1.0, -1.0, 0.0]
+            - neg2: [-1.0, 1.0, 0.0]
```

Comment:
should it not be neg1?

Reply:
It's on purpose since I want all moments to be non-zero in the test

Reply:
Good then!